logic/disk.js

```diff
@@ -15,6 +15,7 @@ const DEFAULT_ADVANCED_SETTINGS = {
   incomingConnections: false,
   cacheSizeMB: 450,
   mempoolFullRbf: false,
+  rest: true,
```

Comment:
We should set this to `false`, so that by default a user who first installs the app isn't unwittingly exposing themselves to privacy/DoS risks. A user can then decide to accept the risks if they want to enable it.

Reply:
Makes sense being more secure by default, changed and squashed in fdb79222a758929278aab08e3879e83c4b70dda5